tests/odm/documents/test_init.py

```diff
@@ -59,6 +60,23 @@ async def test_flagged_index_creation():
     }
 
 
+async def test_flagged_index_creation_with_alias():
```

Comment:
That's great that you made a test for this!

Reply:
Never commit without tests :D 

(except when I added the typing, never tested type before in python :))